qutebrowser/browser/commands.py

```diff
@@ -832,6 +833,62 @@ def paste(self, sel=False, tab=False, bg=False, window=False):
             self._open(url, tab, bg, window)
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
+                       completion=[usertypes.Completion.tab])
+    def buffer(self, index=None):
+        """Select tab by index or url/title best match, focus window if
+        necessary.
+
+        Args:
+            index: The win_id/index of the tab to focus. Or a substring
+                   in which case the closest match will be focused.
+        """
+        if not index or not len(index):
```

Comment:
Empty strings are false in Python, so checking `if not index:` here is enough.

On the other hand, why not just make `index` mandatory by removing the `=None` default?